recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = 'tvb-framework' %}
+{% set version = '1.5.5' %}
+{% set sha256 = "81c6b293bff080f4b52ec2c9ece5a3e109c951eb9dd42b1d90aed00589d4828b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - pip
+    
+  run:
+    - python
+    - beautifulsoup4
+    - cfflib
+    - cherrypy
+    - formencode
+    - genshi
+    - h5py
+    - Networkx
```

Comment:
In conda all the packages names are lower case. This is not a big deal and you can change it later. (That is a lot of dependencies BTW, can you double check if that is correct?)

Reply:
Thanks this is typos is networkx not Networkx